### ISSUE TITLE
`viz/colors` seems not to handle :keyword specified fields

### ISSUE THREAD

**Issue by ezmiller** [scicloj/viz.clj#7]
I came across this problem and haven't had a chance to look into it deeply, but it seems like a small bug. 

Here's a plot:

```
(-> messages-with-features
    (tc/group-by [:month :year])
    (tc/aggregate {:num-messages tc/row-count})
    (tc/order-by :month)
    (viz/data)
    (viz/type ht/line-chart)
    (viz/x :month)
    (viz/y :num-messages)
    (viz/color :year)
    (viz/viz)
    )
```
This plot yielded an error:
> Vega Spec error: TypeError: Cannot use 'in' operator to search for 'scale' in year

But if I changed the `:year` specified in `viz/colors` to `"year"`, the plot generation works.

**Comment by ashimapanjwani** [comment]
Thanks for pointing this out @ezmiller. Will look into it

**Comment by daslu** [comment]
Thanks.

This is solved in version 0.1.1.
